scripts/overcloud-database-recover.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -euE
```

Comment:
I see that a bit of inconsistency, setting some things with their name (-o) and by abbreviation, but whatever.

Reply:
I don't think pipefail has a short flag, so this is quite common.